june/17/arpitupadhyay

```diff
@@ -0,0 +1,8 @@
+# Yesterday
+- Worked on the assigned issue (User Profile Page)
+-Tried learning about Mongoose and databases in general, learning how to access them.
+# Today
+- Working on the same issue.
+-Learning more about Models Schema and database integration into project code.
+# Blockers
+- Having a hard time grasping the above mentioned topics.
```

Comment:
try reading through code, or copy-paste one section in a separate file and play with it.